test/isStringUppercase.test.js

```diff
@@ -0,0 +1,21 @@
+const { isStringUppercase } = require("../lib");
+
+test("isStringUppercase returns false for single char 'c'", () => {
```

Comment:
Hey @v-chmykov ! Awesome work on this. We are trying to move towards wrapping all of our tests into a single "describe" block. See other .test.js files for examples! If you resolve this issue, I will happily merge this in! Thanks!!